config.json

```diff
@@ -66,8 +66,51 @@
     {
       "username": "mender-test-bot",
       "secrets": "mender-secrets.json",
-      "bot_features": ["report_open_prs"],
-      "orgs": [ "mendersoftware" ]
+      "bot_features": [
+        "report_open_prs",
+        "ping_reviewers_dependabot"
+      ],
+      "orgs": [ "mendersoftware" ],
+
+      "repos_dependabot" : {
+        "mendersoftware/gui": "mzedel",
+        "mendersoftware/integration": "lluiscampos",
+        "mendersoftware/meta-mender": "lluiscampos",
+        "mendersoftware/mender": "lluiscampos",
+        "mendersoftware/mender-artifact": "lluiscampos",
+        "mendersoftware/mender-cli": "lluiscampos",
+        "mendersoftware/mender-connect": "lluiscampos",
+        "mendersoftware/mender-binary-delta": "lluiscampos",
+        "mendersoftware/mender-dist-packages": "lluiscampos",
+        "mendersoftware/mender-demo-artifact": "lluiscampos",
+        "mendersoftware/mender-convert": "lluiscampos",
+        "mendersoftware/mender-test-containers": "lluiscampos",
+        "mendersoftware/mender-stress-test-client": "lluiscampos",
+        "mendersoftware/integration-test-runner": "lluiscampos",
+        "mendersoftware/mender-gateway": "lluiscampos",
+        "mendersoftware/auditlogs": "kjaskiewiczz",
+        "mendersoftware/create-artifact-worker": "kjaskiewiczz",
+        "mendersoftware/deployments": "kjaskiewiczz",
+        "mendersoftware/deployments-enterprise": "kjaskiewiczz",
+        "mendersoftware/deviceauth": "kjaskiewiczz",
+        "mendersoftware/deviceauth-enterprise": "kjaskiewiczz",
+        "mendersoftware/deviceconfig": "kjaskiewiczz",
+        "mendersoftware/deviceconnect": "kjaskiewiczz",
+        "mendersoftware/devicemonitor": "kjaskiewiczz",
+        "mendersoftware/generate-delta-worker": "kjaskiewiczz",
+        "mendersoftware/inventory": "kjaskiewiczz",
+        "mendersoftware/inventory-enterprise": "kjaskiewiczz",
+        "mendersoftware/iot-manager": "kjaskiewiczz",
+        "mendersoftware/mender-configure-module": "kjaskiewiczz",
+        "mendersoftware/monitor-client": "kjaskiewiczz",
+        "mendersoftware/mtls-ambassador": "kjaskiewiczz",
+        "mendersoftware/reporting": "kjaskiewiczz",
+        "mendersoftware/tenantadm": "kjaskiewiczz",
+        "mendersoftware/useradm": "kjaskiewiczz",
+        "mendersoftware/useradm-enterprise": "kjaskiewiczz",
+        "mendersoftware/workflows": "kjaskiewiczz",
+        "mendersoftware/workflows-enterprise": "kjaskiewiczz"
```

Comment:
One idea here, if we want to ensure we cover all the repos, is to set a default (Lluis or Krzysztof) and then only list the repos which should have a different reviewer.

Reply:
Originally I wanted something like that: define the backend, client and frontend maintainers and then map repos to team. But I realized that the maintenance effort would be the same as we sill need to add repos to one of the groups any time we have a new repo. So I concluded in having a directy repo to maintainer list.